examples/gym/solve_minigrid.py

```diff
@@ -0,0 +1,155 @@
+import logging
+import time
+
+import numpy as np
+from sqlalchemy import create_engine
+
+import gym
+
+from gym_minigrid.wrappers import *
```

Comment:
i know this is how they say to import, but it's generally not a good practice because you don't know what you're importing and might have naming collisions...

Reply:
Actually this is not necessary, it was my first attempts to see how the minigrid works. We don't necessarily need the wrappers, but if we stick to RGB images as inputs then the `RGBImgPartialObsWrapper` wrapper is necessary. The default image in minigrid is given as cell position on the grid